Sources/bazel-rest-cache/Cache.swift

```diff
@@ -0,0 +1,181 @@
+import Foundation
+import Kitura
+import LoggerAPI
+import SwiftRedis
+
+final class Cache {
+
+    init(router: Router) {
+        addRoutes(router: router)
+    }
+
+    // MARK: Private
+
+    private let byteCountFormatter: ByteCountFormatter = {
+        let formatter = ByteCountFormatter()
+        formatter.allowedUnits = [
+            .useKB,
+            .useMB,
+            .useGB,
+        ]
+        return formatter
+    }()
+
+    private func addRoutes(router: Router) {
```

Comment:
@kersson All I really did was refactor this into a class, and created identical sets of routes for each 'prefix'.